modules/data.remote/R/soilgrids.R

```diff
@@ -0,0 +1,224 @@
+##' @title soilgrids.soc.extract
+##' @name  soilgrids.soc.extract
+##' 
+##' @param site_info A BETYdb site info dataframe containing at least each site ID, sitename, 
+##' latitude, longitude, and time_zone. e.g. c(site_qry$id, site_qry$sitename, site_qry$lon,
```

Comment:
why would time zone  be needed to extract soil information that doesn't have a time component? Also, why are both site ID and name required? Seems like the code would be more portable if only one sort of identifier was required.

Reply:
Follow up -- looked through the code and can't find ANYPLACE time_zone and sitename are actually used. Would recommend dropping these as required arguments

Reply:
TZ removed - I dont remember why i did that - I think perhaps related to how I thought it was needed for storing in BETYdb...